holoviews/ipython/magics.py

```diff
@@ -337,25 +344,48 @@ def output(self, line, cell=None):
             print("\nFor help with the %output magic, call %output?")
             return
 
-        restore_copy = OrderedDict(OutputMagic.options.items())
+        # Make backup of previous options
+        prev_backend = Store.current_backend
+        prev_renderer = Store.renderers[prev_backend]
+        prev_mode = prev_renderer.mode
+        prev_params = {k: v for k, v in prev_renderer.get_param_values()
+                       if k in self.render_params}
+        prev_restore = dict(OutputMagic.options)
```

Comment:
Maybe ``prev`` is confusing as it suggests the backend is changing (which it might not). Maybe ``initial_backend``? Implement this if you like the suggestion, it isn't something worth holding up the PR for.